plugin/core/rpc.py

```diff
@@ -44,7 +44,7 @@ def outgoing_notification(self, method: str, params: Any) -> None:
         pass
 
     @abstractmethod
-    def incoming_response(self, request_id: int, params: Any) -> None:
+    def incoming_response(self, request_id: int, params: Any, blocking: Optional[bool] = False) -> None:
```

Comment:
Since the default is already `False`, it's IMO better for this argument to be of type `bool` than of type `Optional[bool]`.

Reply:
In fact there should not be a default argument, because `outgoing_request` doesn't have a default argument either. It's more symmetric if the signatures stay somewhat the same.